notebooks/iatraoui.ipynb

```diff
@@ -237,8 +293,26 @@
     "        _type_: A list of the RGB values contained in the data_array and their corresponding classes\n",
     "    \"\"\"\n",
     "\n",
-    "    ...  # TODO\n",
-    "    raise NotImplementedError\n"
+    "    data_array = data_array.stack(z=(\"x\", \"y\"))\n",
+    "    burnt_polygons = burnt_polygons.reshape(1830**2)\n",
+    "    burnt_polygons = burnt_polygons.astype(\"int64\")\n",
+    "\n",
+    "    classified_rgb_rows = xr.Dataset(\n",
+    "        {\n",
+    "            \"reflectance\": (\n",
+    "                (\"band\", \"z\"),\n",
+    "                data_array.data,\n",
```

Comment:
You don't need to access the underlying data of a DataArray to construct a Dataset.
You can just pass the DataArray directly as a value to the `"reflectance"` key in the dict!

This way, you don't need to specify the `coords` parameter, as they are already contained in the DataArray

Note that you might need to wrap the "feature ids" Numpy array into a DataArray with the `z` dim.

Example:


```python
In [17]: da = xr.DataArray(data=arr, coords={"band": ["a", "b", "c", "d"], "z": list(range(0,4*60,60))})

In [18]: da
Out[18]: 
<xarray.DataArray (band: 4, z: 4)>
array([[ 0,  1,  2,  3],
       [ 4,  5,  6,  7],
       [ 8,  9, 10, 11],
       [12, 13, 14, 15]])
Coordinates:
  * band     (band) <U1 'a' 'b' 'c' 'd'
  * z        (z) int32 0 60 120 180

In [19]: xr.Dataset({"reflectance": da, "feature_id": xr.DataArray([3,2,2,3], dims="z")})
Out[19]:
<xarray.Dataset>
Dimensions:      (band: 4, z: 4)
Coordinates:
  * band         (band) <U1 'a' 'b' 'c' 'd'
  * z            (z) int32 0 60 120 180
Data variables:
    reflectance  (band, z) int32 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15
    feature_id   (z) int32 3 2 2 3
```